lib/peridio/rat/tunnel.ex

```diff
@@ -152,7 +152,8 @@ defmodule Peridio.RAT.Tunnel do
             Logger.debug("Tunnel #{state.id} bringing up interface #{state.interface.id}")
             interface_up(state)
 
-          _error ->
+          error ->
+            Logger.error("Tunnel #{state.id} error configuring interface #{error}")
```

Comment:
Should inspect that error, it can be a tuple which, among other terms, will cause the attempt to log itself to raise an exception.

```suggestion
            Logger.error("Tunnel #{state.id} error configuring interface #{inspect error}")
```